### DISCUSSION TITLE
quarto book with docx and pdf format + biblatex citation

### DISCUSSION THREAD

**Discussion by iusgit** [quarto-dev/quarto-cli#2600] [Q&A]
Hi,

I'm creating a book project with "pdf" and "docx" output. Since I prefer biblatex citation mode for "pdf", I added the option `cite-method: biblatex` to the pdf YAML block. Activating biblatex leads to a hard-coded part in the partials that executes `printbibliograpy` in Latex. For "docx" I added a chapter:
```
# Referenzen {.unnumbered}

::: {#refs}
:::
```
That leads to 2 chapters for literature... One from hardcoded printbibliography, one empty from quarto.

So I thought I make that conditional:
```
::::: {.content-hidden unless-format="docx"}
# Referenzen {.unnumbered}

::: {#refs}
:::

:::::
```

But that doesn't help, "docx" is OK, but "pdf" still has the 2 chapters.

Next step would be to go into the partials, but I wanted to know if somebody has a solution?

**Comment by iusgit** [comment]
May solution is like mentioned above to comment out `\printbibliography` in file `biblio.tex`as a partial and use biblatex with option `heading=none`to avoid the superfluous chapter:
```
# Referenzen {.unnumbered}

::: {#refs}
:::

::: {.content-visible when-format="pdf"}

\printbibliography[heading=none]

:::
```

**Reply by fuhrmanator** [reply]
Can you share how you defined the partials? I'm on Windows and I get an error trying to set my own partial in the metadata (see https://github.com/quarto-dev/quarto-cli/discussions/1555)
